sys/dev/xdma/xdma.h

```diff
@@ -95,19 +95,24 @@ struct xchan_buf {
 	bus_dmamap_t			map;
 	uint32_t			nsegs;
 	uint32_t			nsegs_left;
-	vm_offset_t			vaddr;
+	vm_pointer_t			vaddr;
 	vm_offset_t			paddr;
 	vm_size_t			size;
 };
 
+typedef union {
+	uintptr_t		vaddr;
+	bus_addr_t		addr;
```

Comment:
These are somewhat confusing names...

Reply:
Perhasp `vaddr` and `bus_addr` are clearer.

Reply:
Although this means that probably we want to change the naming also in bus_dma.h as it somewhat mirrors that one.